### DISCUSSION TITLE
SVG not rendering in app

### DISCUSSION THREAD

**Discussion by halitsever** [community/community#3693] [Mobile] [answered]
Android Version : 9
Chrome Version: 89.04.4389.105
Github app version: 1.7.5

**Github App:**                 &nbsp;  &nbsp;  &nbsp;  &nbsp;  &nbsp;  &nbsp;  &nbsp;  &nbsp;  &nbsp;  &nbsp;  &nbsp;  &nbsp;  &nbsp;  &nbsp;  &nbsp;  &nbsp;  &nbsp;  &nbsp;  &nbsp;  &nbsp;  &nbsp;  &nbsp;  &nbsp;  &nbsp;  &nbsp;  &nbsp;  &nbsp;  &nbsp;  &nbsp;  &nbsp;  &nbsp;  &nbsp;  &nbsp;  &nbsp;  &nbsp;  &nbsp;  &nbsp;  &nbsp;  &nbsp;  &nbsp;  &nbsp;  &nbsp;  &nbsp;  &nbsp;  &nbsp;  &nbsp;  &nbsp;                                                        **Chrome:**
<img height="600" src="https://i.ibb.co/rpVHqd3/Inked-Screenshot-20210404-044920-LI.jpg"> <img height="600" src="https://i.ibb.co/S01rKXP/Whats-App-Image-2021-04-04-at-18-21-00.jpg">

**Comment by brianlovin** [comment, marked as answer]
Thanks for the report here @halitsever – because of the complex nature of that SVG (internal style tags and html) we aren't able to render this natively right now. We have some work in progress to improve our rendering capabilities for markdown files, which should hopefully improve this case 😊
